chalicelib/responseobjects/hca_response_v5.py

```diff
@@ -122,26 +125,52 @@ class ManifestResponse(AbstractResponse):
     """
     Class for the Manifest response. Based on the AbstractionResponse class
     """
+
     def return_response(self):
-        return self.apiResponse
+        es_search = self.es_search
+
+        def field_mapper():
+            for hit in es_search.scan():
+                hit_dict = hit.to_dict()
+                mapped_dict = {}
+                for friendly_name_key, es_name_value in self.mapping.iteritems():
```

Comment:
@jvlasblom This is the portion @hannes-ucsc was referring to. You can probably iterate first over `self.manifest_entries` and check if the value is present in `self.mapping`